libs/media-messages/schemas/payloads/media/media-service-asyncapi.yml

```diff
@@ -79,7 +79,7 @@ channels:
     bindings:
       amqp:
         queue:
-          name: entity:deleted
+          name: inbox
```

Comment:
Just to make sure, should this also be changed, even though this is an event and
not a command?

Reply:
Yes, the only consumer is in this case the media service so this is fine. Also generally: this is an AsyncAPI "suggestion" on how to name a queue. The defining service of the event makes a suggestion on how the consumer should name its queue.